distribution/docs/src/main/resources/content/_architectures/_securityFramework/realms.adoc

```diff
@@ -49,39 +55,37 @@ They are passed in the `AuthorizationInfo` of the Subject along with the permiss
 
 |org.codice.ddf.security.filter.websso.WebSSOFilter
 |This filter is the main security filter that works with a number of handlers to protect a variety of web contexts, each using different authentication schemes and policies.
-
-|org.codice.ddf.security.handler.saml.SAMLAssertionHandler
-|This handler is executed by the WebSSOFilter for any contexts configured to use it.
-This handler should always come first when configured in the Web Context Policy Manager, as it provides a caching capability to web contexts that use it.
-The handler will first check for the existence of an HTTP Authorization header of type SAML, whose value is a Base64 + deflate SAML assertion.
-If that is not found, then the handler will check for the existence of the deprecated `org.codice.websso.saml.token` cookie with the same value.
-Failing that, it will check for a JSESSIONID cookie to use as a reference to a cached assertion.
-If the JSESSIONID is valid, the `SecurityToken` will be retrieved from the cache.
+Its job is to attach an Authentication Token to the request.
```

Comment:
we could get this down to `It attaches an Authentication Token to the request by either checking the session or calling the configured Security Handlers.`